test/web3/Main.purs

```diff
@@ -34,26 +34,26 @@ main =
     p <- liftEffect $ httpProvider "http://localhost:8545"
     join
       $ runSpecT cfg [ consoleReporter ] do
```

Comment:
why are these being commented out?

Reply:
sorry, I was testing error on my computer, will clean this up